dpm/api/env.py

```diff
@@ -15,38 +19,48 @@ class Env:
     dpm_initialized: bool = False
     secrets_initialized: bool = False
 
-    @staticmethod
```

Comment:
I think this could effect a large number of our current use cases; so we'd need to lock their version number, or come up with a minor workaround. 

Moving away from statics is a goal for sure, so this effort is appreciated.